packages/components/src/FormatDate/FormatDate.test.tsx

```diff
@@ -2,17 +2,51 @@ import React from "react";
 import renderer from "react-test-renderer";
 import { cleanup } from "@testing-library/react";
 import { CivilDate } from "@std-proposal/temporal";
-import { FormatDate } from "./FormatDate";
+import { FormatDate, strFormatDate } from "./FormatDate";
 
 afterEach(cleanup);
 
-Object.entries({
-  CivilDate: new CivilDate(2020, 2, 26),
-  ISO8601DateString: "2019-03-30T00:45",
-  Date: new Date("2019-03-30T00:45"),
-}).forEach(([inputType, value]) => {
-  it(`renders a FormatDate from ${inputType}`, () => {
-    const tree = renderer.create(<FormatDate date={value} />).toJSON();
-    expect(tree).toMatchSnapshot();
+describe("Different date values", () => {
+  const dates = {
+    CivilDate: new CivilDate(2020, 2, 26),
+    ISO8601DateString: "2019-03-30T00:45",
+    Date: new Date("2019-03-30T00:45"),
+  };
+
+  const mockDateResult = [`Feb 26, 2020`, `Mar 30, 2019`, `Mar 30, 2019`];
```

Comment:
Repeat date here

Reply:
It's looping through 3 tests with different variables but return the exact date 😛

Reply:
Oh understood!